.github/workflows/continuous-delivery.yaml

```
@@ -30,16 +30,16 @@ jobs:
         run: npm ci --ignore-scripts --no-audit --no-progress --prefer-offline
       - name: Build
         run: npm run build
-        id: build_package
       - env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN_DOTTBOTT }}
+        id: release_package
```

This can simply match the name.

```suggestion
        id: release
```